mozdownload/scraper.py

```diff
@@ -55,6 +55,10 @@
                            'win32': 'exe',
                            'win64': 'exe'}
 
+DEFAULT_BRANCHES = {'b2g': 'mozilla-central',
+                    'firefox': 'mozilla-central',
+                    'thunderbird': 'comm-central'}
```

Comment:
I don't like that we hard-code the application names twice. There is already the `APPLICATIONS` variable available. We should be able to change it to a hash with the application name as key and the appropriate branch as value.